src/com/linkedin/parseq/Task.java

```diff
@@ -1025,6 +1028,43 @@
     return callable("callable: " + _taskDescriptor.getDescription(callable.getClass().getName()), callable);
   }
 
+  /**
+   * Creates a new task that's value will be set to the value returned
+   * from the CompletionStage.
+   *
+   * Returned task will fail if callable passed in as a parameter throws
+   * an exception.
+   *
+   * @param <T> the type of the return value for this task
+   * @param desc description of the task, it will show up in a trace
+   * @param callable the callable to execute when this task is run
+   * @return the new task that will invoke the callable and complete CompletionStage with its result
```

Comment:
This comment is a bit misleading. It suggests that task will complete CompletionStage. Can you rephrase e.g.:
"the new task that will invoke the callable and complete with result of returned CompletionStage"